packages/augur-tools/src/libs/contract-api.ts

```diff
@@ -724,4 +693,46 @@ export class ContractAPI {
     });
   }
 
+  async fundSafe(safe=undefined) {
+    safe = safe || await this.getOrCreateSafe();
+
+    await this.faucet(new BigNumber(1e21));
+    await this.transferCash(safe, new BigNumber(1e21));
+
+    let status: string;
+    for (let i = 0; i < 10; i++) {
+      status = await this.getSafeStatus(safe);
+      if (status !== GnosisSafeState.WAITING_FOR_FUNDS) {
+        break;
+      }
+      await sleep(2000);
+    }
+
+    await sleep(10000);
```

Comment:
fyi this can probably be dropped. I put it in during debug but never took it out.